app/models/miq_region.rb

```diff
@@ -99,6 +99,10 @@ def find_master_server
 
   cache_with_timeout(:my_region) { find_by(:region => my_region_number) }
 
+  cache_with_timeout(:my_zone_names) do
+    Zone.in_region(my_region_number).pluck(:name)
+  end
```

Comment:
Alternatively, we can just use `my_region` as the cache and store in the object:

```ruby
  def zone_names do
    @zone_names ||= Zone.in_region(region_number).pluck(:name)
  end
```

Reply:
Yes, I'd rather not introduce a new cache when one already exists.

Reply:
I did that - using `MiqRegion.zones` cache

Reply:
moved this stuff to another PR